projects/admin/src/app/record/detail-view/item-detail-view/item-transactions/item-transactions.component.html

```diff
@@ -22,8 +22,8 @@
         <b>{{ 'Borrowed by' | translate }}</b>
       </div>
       <div class="card-body">
-        <div class="row">
-          <div class="col-5 font-weight-bold">{{ 'Patron' | translate }}</div>
+        <div class="row mb-2">
+          <div class="col-4 font-weight-bold">{{ 'Patron' | translate }}</div>
           <div class="col-4 font-weight-bold">{{ 'Renewal' | translate }}</div>
           <div class="col-3 font-weight-bold">{{ 'Due date' | translate }}</div>
         </div>
```

Comment:
4 + 4 + 3, It's still missing 1 col ? Is it conscious ?

Reply:
Look at line 57